code/model/DMSDocument.php

```diff
@@ -1138,7 +1132,13 @@ public function onBeforeWrite()
                 $this->expireAtDate($this->ExpireAtDate, false);
             } else {
                 $this->clearExpiry(false);
-            } //clear all previous settings
+            } // clear all previous settings
```

Comment:
Code style (caps lock just to keep the consistency between these files @fspringveldt )